barretenberg/cpp/src/barretenberg/dsl/acir_format/acir_format_mocks.cpp

```diff
@@ -0,0 +1,118 @@
+#include "acir_format.hpp"
+
+acir_format::AcirFormatOriginalOpcodeIndices create_empty_original_opcode_indices()
+{
+    return acir_format::AcirFormatOriginalOpcodeIndices{
```

Comment:
Try it out, but I'd be surprised if you have to be explicit here. If you do `return acir_format::AcirFormatOriginalOpcodeIndices()` it should be equivalent. Objects have default constructors.